doc/source/serve/doc_code/vllm_openai_example.py

```diff
@@ -0,0 +1,106 @@
+# __serve_example_begin__
+from typing import Dict, Optional, List
+import logging
+from ray import serve
+from fastapi import FastAPI
+from starlette.requests import Request
+from starlette.responses import StreamingResponse, Response, JSONResponse
+
+from vllm.engine.arg_utils import AsyncEngineArgs
+from vllm.engine.async_llm_engine import AsyncLLMEngine
+from vllm.entrypoints.openai.cli_args import make_arg_parser
+from vllm.entrypoints.openai.protocol import ChatCompletionRequest, ChatCompletionResponse, ErrorResponse
+from vllm.entrypoints.openai.serving_chat import OpenAIServingChat
+from vllm.entrypoints.openai.serving_engine import LoRAModulePath
```

Comment:
I typically try to pull out any ray-related imports into a separate block, so it's clear what Ray dependencies are needed for the example.

```suggestion
import logging
from typing import Dict, Optional, List

from fastapi import FastAPI
from starlette.requests import Request
from starlette.responses import StreamingResponse, Response, JSONResponse

from vllm.engine.arg_utils import AsyncEngineArgs
from vllm.engine.async_llm_engine import AsyncLLMEngine
from vllm.entrypoints.openai.cli_args import make_arg_parser
from vllm.entrypoints.openai.protocol import ChatCompletionRequest, ChatCompletionResponse, ErrorResponse
from vllm.entrypoints.openai.serving_chat import OpenAIServingChat
from vllm.entrypoints.openai.serving_engine import LoRAModulePath

from ray import serve
```